opencensus/trace/propagation/grpc_trace_bin.h

```diff
@@ -43,6 +44,12 @@ SpanContext FromGRPCTraceBinHeader(absl::string_view header);
 // Returns a value for the grpc-trace-bin header.
 std::string ToGRPCTraceBinHeader(const SpanContext& ctx);
 
+constexpr int kGRPCTraceBinHeaderLen = 29;
```

Comment:
A comment to explain the size:
1 (version) + 1 (delimiter) + ...

Reply:
Done.